libs/secrets/src/aes.h

```diff
@@ -26,30 +26,31 @@ namespace concord::secretsmanager {
 
 class IAesMode {
  public:
-  IAesMode(const KeyParams& params, const std::string& a_info = "") : params_{params}, additional_info(a_info) {}
+  IAesMode(const KeyParams& params, const uint32_t tagLengthBits = 128)
+      : params_{params}, tag_length_in_bits(tagLengthBits) {}
   virtual std::vector<uint8_t> encrypt(std::string_view input) = 0;
   virtual std::string decrypt(const std::vector<uint8_t>& cipher) = 0;
   const KeyParams getKeyParams() { return params_; }
-  const std::string getAdditionalInfo() { return additional_info; }
```

Comment:
Should we consider keeping this in the interface even though we may not be using it since it is meant for sending additional information in the original response.

Reply:
Since i removed the additional_info from the interface, hence removed the method as well. As discussed with Galya, we keep this as null and mostly used in case of any error which is dealt in other class.. We can still discuss and add this field for future purpose.